.editorconfig

```diff
@@ -0,0 +1,11 @@
+# editorconfig.org
```

Comment:
i think this is a file generated by your editor, it should not be included by this repo :)